### DISCUSSION TITLE
Sometimes code coloring is not working for a part of .java file

### DISCUSSION THREAD

**Discussion by motacapla** [community/community#11591] [Other Feature Feedback, Questions, & Ideas]
Hi 👋 
Recently I have faced the issue in the following ```files``` page:
```
https://github.com/<branch_path>/pull/<pr_num>/files
```

Most of files written in .java have own colors so that it's easy for me to review, however, sometimes there is a file without the colors.
These co-exist in 1 PR, e.g., A.java has color but B.java doesn't have color.

Do you have any ideas about this issue?

**Comment by Leollaine** [comment]
https://github.com/<branch_path>/pull/<pr_num>/files
